### PR TITLE
Fix createCoordSeq + test

### LINKED ISSUE #24: segmentation fault in Point constructor
I'm getting a seg fault when I'm creating a point with integers

```julia
Point(2, 3)
```

results in 

signal (11): Segmentation fault
while loading no file, in expression starting on line 0
unknown function (ip: 0xffffffffffffffff)
Type at /home/omri/.julia/v0.5/LibGEOS/src/geos_types.jl:10
unknown function (ip: 0x7fd1367347cc)
jl_call_method_internal at /build/julia-c0WfgE/julia-0.5.0/src/julia_internal.h:189 [inlined]
jl_apply_generic at /build/julia-c0WfgE/julia-0.5.0/src/gf.c:1942
do_call at /build/julia-c0WfgE/julia-0.5.0/src/interpreter.c:66
eval at /build/julia-c0WfgE/julia-0.5.0/src/interpreter.c:190
jl_toplevel_eval_flex at /build/julia-c0WfgE/julia-0.5.0/src/toplevel.c:558 [inlined]
jl_toplevel_eval at /build/julia-c0WfgE/julia-0.5.0/src/toplevel.c:580
jl_toplevel_eval_in_warn at /build/julia-c0WfgE/julia-0.5.0/src/builtins.c:590
eval at ./boot.jl:234
unknown function (ip: 0x7fd34530f67f)
jl_call_method_internal at /build/julia-c0WfgE/julia-0.5.0/src/julia_internal.h:189 [inlined]
jl_apply_generic at /build/julia-c0WfgE/julia-0.5.0/src/gf.c:1942
eval_user_input at ./REPL.jl:64
unknown function (ip: 0x7fd13672cdb6)
jl_call_method_internal at /build/julia-c0WfgE/julia-0.5.0/src/julia_internal.h:189 [inlined]
jl_apply_generic at /build/julia-c0WfgE/julia-0.5.0/src/gf.c:1942
macro expansion at ./REPL.jl:95 [inlined]
#3 at ./event.jl:68
unknown function (ip: 0x7fd136729f2f)
jl_call_method_internal at /build/julia-c0WfgE/julia-0.5.0/src/julia_internal.h:189 [inlined]
jl_apply_generic at /build/julia-c0WfgE/julia-0.5.0/src/gf.c:1942
jl_apply at /build/julia-c0WfgE/julia-0.5.0/src/julia.h:1392 [inlined]
start_task at /build/julia-c0WfgE/julia-0.5.0/src/task.c:253
unknown function (ip: 0xffffffffffffffff)
Allocations: 2023301 (Pool: 2022391; Big: 910); GC: 1
Segmentation fault

### PR DESCRIPTION
**Pull request by OmriRoames**
fixed the typo and added some tests. This also fixes #24

### CODE REVIEW

**Comment by OmriRoames** [comment]
OK, ready for review

File: test/test_geos_functions.jl [hunk 1]
```diff
@@ -80,10 +80,23 @@ LibGEOS.destroyGeom(expected_)
...
+@fact LibGEOS.getSize(cs_) --> 1
+@fact LibGEOS.getDimensions(cs_) --> 2
+@fact LibGEOS.getCoordinates(cs_) --> [5, 3]
+
+cs_2 = LibGEOS.createCoordSeq([5.0, 3.0])
+@fact LibGEOS.getSize(cs_2) --> 1
+@fact LibGEOS.getDimensions(cs_2) --> 2
+@fact LibGEOS.getCoordinates(cs_2) --> [5.0, 3.0]
+
+cs_3 = LibGEOS.createCoordSeq([[5.0, 3.0], [1.0, 2.0], [1.0, 3.0]])
```
Comment (yeesian) [inline test/test_geos_functions.jl]: I don't think this works on julia v0.4. Can you change it to `Vector{Float64}[[5.0, 3.0], [1.0, 2.0], [1.0, 3.0]]` instead?

Comment (OmriRoames) [inline test/test_geos_functions.jl]: done

**Comment by yeesian** [comment]
I don't know how to make sense of the failures on travis and appveyor. Allow me to check with @visr and @tkelman first.

**Comment by tkelman** [comment]
guessing, but win64 might be an integer size mismatch - `Clong` is 32 bits there

**Comment by visr** [comment]
Testing this branch locally on win64 with julia 0.5, it segfaults intermittently. When it does, it is on either of these two lines:

https://github.com/JuliaGeo/LibGEOS.jl/blob/2dacd7977e168b5a484797353d921e261e117b10/test/test_geos_functions.jl#L240
https://github.com/JuliaGeo/LibGEOS.jl/blob/2dacd7977e168b5a484797353d921e261e117b10/test/test_geos_functions.jl#L272

I tried some things like adding a `GEOSFree` call to the result of `GEOSGeomToWKT`, but it doesn't help, or is not enough:
```julia
function geomToWKT(geom::Ptr{GEOSGeometry})
    ptr = GEOSGeomToWKT(geom)
    wkt = unsafe_string(ptr)
    GEOSFree(convert(Ptr{Void}, ptr))
    wkt
end
```

This error message seems to point it to string conversion functions:
```
Exception: EXCEPTION_ACCESS_VIOLATION at 0x770beef1 -- RtlAnsiStringToUnicodeString at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
while loading C:\Users\visser_mn\.julia\v0.5\LibGEOS\test\test_geos_functions.jl, in expression starting on line 272
RtlAnsiStringToUnicodeString at C:\WINDOWS\SYSTEM32\ntdll.dll (unknown line)
free at C:\WINDOWS\system32\msvcrt.dll (unknown line)
ZN4llvm23SmallVectorTemplateBaseINS_28DWARFAbbreviationDeclaration13AttributeSpecELb0EE4growEy at C:\bin\julia-0.5\bin\LLVM-3.7.dll (unknown line)
ZN4llvm28DWARFAbbreviationDeclaration7extractENS_13DataExtractorEPj at C:\bin\julia-0.5\bin\LLVM-3.7.dll (unknown line)
ZN4llvm31DWARFAbbreviationDeclarationSet7extractENS_13DataExtractorEPj at C:\bin\julia-0.5\bin\LLVM-3.7.dll (unknown line)
ZN4llvm16DWARFDebugAbbrev7extractENS_13DataExtractorE at C:\bin\julia-0.5\bin\LLVM-3.7.dll (unknown line)
ZN4llvm12DWARFContext26getLineInfoForAddressRangeEyyNS_19DILineInfoSpecifierE at C:\bin\julia-0.5\bin\LLVM-3.7.dll (unknown line)
ZN4llvm20DWARFUnitSectionBase5parseERNS_12DWARFContextERKNS_12DWARFSectionE at C:\bin\julia-0.5\bin\LLVM-3.7.dll (unknown line)
ZN4llvm12DWARFContext23getCompileUnitForOffsetEj at C:\bin\julia-0.5\bin\LLVM-3.7.dll (unknown line)
ZN4llvm12DWARFContext25getInliningInfoForAddressEyNS_19DILineInfoSpecifierE at C:\bin\julia-0.5\bin\LLVM-3.7.dll (unknown line)
```

**Comment by yeesian** [comment]
@visr if it's not too inconvenient, can you help test this branch on win64 with julia 0.5 again? It'll be good to have one last fix for this before we drop support for 0.4, and start on a new tag targeting 0.6.

**Comment by visr** [comment]
Ahh managed to isolate the issue now, see the added commit. All green now.

**Comment by yeesian** [comment]
Nice, thanks! (:

**Pull request merged**